active_record-humanized_enum.gemspec

```
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
 
   spec.files       = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
 
-  spec.add_dependency 'railties'
+  spec.add_dependency 'railties', ">= 7.0"
```

It works with Rails versions up to 4.

```suggestion
  spec.add_dependency 'railties', '>= 4.0'
```

---

commited. Thanks for the clarification